diff --git a/stm32f469i-disco.c b/stm32f469i-disco.c
index d4d0909831..03f823f288 100644
--- a/stm32f469i-disco.c
+++ b/stm32f469i-disco.c
@@ -127,7 +127,7 @@ int main(void)
 
 	int i;
 
-	mpu_config(0x0);
+	/* mpu_config(0x0); */
 
 	if (*FLASH_CR & FLASH_CR_LOCK) {
 		*FLASH_KEYR = 0x45670123;
@@ -229,7 +229,7 @@ int main(void)
 	usart_setup(usart_base, 45000000);
 	usart_putch(usart_base, '.');
 
-	SYSCFG_MEMRMP = 0x4;
+	/* *SYSCFG_MEMRMP = 0x4; */
 
 	start_kernel();
 
